response_operations_ui/controllers/sample_controllers.py

```diff
@@ -33,3 +33,20 @@ def upload_sample(short_name, period, file):
                  sample_id=sample_summary['id'])
 
     return sample_summary
+
+
+def get_sample_summary(summary_uuid):
+    logger.debug('Getting sample summary', summary_uuid=summary_uuid)
+
+    url = f'{app.config["SAMPLE_URL"]}/samples/samplesummary/{summary_uuid}'
+    response = requests.get(url, auth=app.config['SAMPLE_AUTH'])
+
+    try:
+        response.raise_for_status()
+    except (HTTPError, RequestException):
```

Comment:
We should handle 4xx and 5xx errors differently.

Reply:
If we're wanting to do this it might be a wider piece of work across the repo/multiple repos? We probably want to be consistent and there's a lot of places where we're not doing this

Reply:
Yep, i've asked @nicprice for RAD's views on this

Reply:
@JamesGardiner worth raising a TBL for this?